index.js

```diff
@@ -66,79 +72,67 @@ function fastifySwagger (fastify, opts, next) {
     }
 
     swaggerObject.paths = {}
+    for (var route of routes) {
+      const url = formatParamUrl(route.url)
+      const method = route.method.toLowerCase()
```

Comment:
**NOTE:** route method uses a capital case and swagger definition uses lowercase format.